cfme/tests/intelligence/reports/test_validate_chargeback_report_manual.py

```diff
@@ -0,0 +1,56 @@
+# pylint: skip-file
```

Comment:
Really wanted to merge this, but this line shouldn't be here.

Were you seeing flake or pre-commit failures?

Reply:
It got copied over from test_manual.py when I created test_validate_chargeback_report_manual.py. I've removed it from both.